### PR TITLE
Lexorin now (approximately) 50% less deadly

### PR DESCRIPTION
**Pull request by Mqiib**
# General Documentation

### Intent of your Pull Request

Currently lexorin only needs to do ~55 damage to knock you out, which is half of what other toxins need to do. This adjusts the damage it does to 3/tick instead of 5 as well as reduce the breath loss stacks by half so it doesnt last for-fucking-ever after it's done metabolizing and kill you anyways.
 
### Why is this change good for the game?

Lexorin is INCREDIBLY deadly as it is. It's still deadly here, but not "you have 30 seconds before you're dead" deadly.

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

Lexorin does less oxyloss damage (5->3) and only add 1 breath loss stack

### What should players be aware of when it comes to the changes your PR is implementing?

Lexorin isn't an absolute death sentence without immediate medical aid now.

### What general grouping does this PR fall under? 

Chemistry balance

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

Lexorin does less oxyloss damage and half as many breath loss stacks.

# Changelog

:cl:  
 
tweak: Lexorin oxyloss from 5 to 3 and breathloss stacks from 2 to 1 per cycle

/:cl:

### CODE REVIEW

**Comment by SomeguyManperson** [comment]
that exception is there to prevent taking oxyloss from the losebreath caused by lexorin, lexorin itself deals like 5 oxyloss on its own so this would make it slightly MORE lethal 30% of the time

**Comment by Mqiib** [comment]
> that exception is there to prevent taking oxyloss from the losebreath caused by lexorin, lexorin itself deals like 5 oxyloss on its own so this would make it slightly MORE lethal 30% of the time

Whoop, missed that because hot ice is for some reason between lexorin and its on_mob_life. I'll adjust that part instead.

**Comment by UselessTheremin** [comment]
an actual good ided pr also
# good god those commits my eyes are burning and IM bad at git

**Comment by Mqiib** [comment]
> an actual good ided pr also
> 
> # good god those commits my eyes are burning and IM bad at git

ARE YOU INTIMIDATED BY MY CODING PROWESS?
Also it's not really an ided.

**Comment by N3D6** [comment]
Merge remote-tracking branch

**Comment by patpol4** [comment]
would the lethal injection syringe still work after this?

**Comment by SomeguyManperson** [comment]
no lethal injection syringe uses lexorin iirc so it won't effect them

**Comment by patpol4** [comment]
> no lethal injection syringe uses lexorin iirc so it won't effect them

Just checked, i was wrong, it doesnt contain lexorin
image

**Review by SomeguyManperson — APPROVED** [review]

**Comment by ynot01** [comment]
lexorin is one of the worst lethal chems
